crypto/rand/drbg_lib.c

```diff
@@ -164,7 +229,14 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
     }
 
     drbg->state = DRBG_READY;
-    drbg->reseed_counter = 1;
+    drbg->reseed_counter = 0;
+
+    if (drbg->reseed_time != 0) {
+        if (drbg->parent == NULL)
+            drbg->reseed_time = time(0);
+        else
+            drbg->reseed_time = drbg->parent->reseed_time;
```

Comment:
It is clear to me that the child DRBG will not reseed more than once per second, even if the parent does, e.g., because RAND_add() is called more than once per second. But I believe that does not matter and I wanted to keep the implementation simple.

Reply:
NULL not 0.  And maybe ```time(&drbg->reseed_time)```

Reply:
Sometimes I just can't hide the fact, that I'm more used to C++. I will stick to your first suggestion, because as a mathematician I prefer the assignment. ;-)

Reply:
And, yes, I kow 0 is out, everybody uses nullptr now ;-)